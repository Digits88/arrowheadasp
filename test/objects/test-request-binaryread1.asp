Test binary read.

Request.TotalBytes = <%=Request.TotalBytes%><br>
Result = Request.BinaryRead(Request.TotalBytes)
    <% Result = Request.BinaryRead(Request.TotalBytes) %><br>
