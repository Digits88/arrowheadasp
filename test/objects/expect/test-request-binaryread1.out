Test binary read.

Request.TotalBytes = 39<br>
Result = Request.BinaryRead(Request.TotalBytes)
    <br>
